apps/prairielearn/src/pages/partials/plr/liveScoreboard.ejs

```diff
@@ -65,6 +65,9 @@
   function updateScores(scores) {
     const tableBody = document.querySelector('#live-results');
     tableBody.innerHTML = '';
+    console.log(scores.length);
```

Comment:
Remove `console.log`s when you're done with em — can do a bigger cleanup after all the functionality is implemented though.